src/labtools/adtools/finder.py

```diff
@@ -50,10 +50,23 @@ def pull_AD(read, barcoded = False, ad_preceder = "GCTAGC",
                 searched_read = re.split(bc_anteceder, roi[ad_length:], maxsplit=1)
                 if len(searched_read) == 2:
                     barcode = searched_read[0][-bclength:]
+                # LT: Increment loss_table if barcode is not found by either flanking sequence
+                else:
+                    loss_table['bc_flanks'] += 1
+            if barcode is not None and len(barcode != bclength:
+                # LT: Increment loss_table if barcode is not the correct length
+                loss_table['bc_length'] += 1
             if barcode == None or len(barcode) != bclength:
+                loss_table['bc_length'] += 1
                 barcode = None
+                # LT: Increment loss_table to show total number of reads without barcodes
+                if barcode == None:
+                loss_table['total_bc_not_found'] += 1
```

Comment:
```suggestion
                    loss_table['total_bc_not_found'] += 1
```

Reply:
Little syntax error here. Also, doublecheck the logic. In the event that barcode is none you will increment `bc_length` AND `total_bc_not_found` which is probably not what you want.